lib/utilities.js

```diff
@@ -213,26 +213,36 @@ utilities._requestSubResource = async function _requestSubResource (request, inf
 
   // Where will we find this resource?
   // if a `context` is available, proxy it to sub resources
-  var subResourceUrl = utilities._buildHref(hrefParameters, request.url.query.context, proxiedValues)
+  const subResourceUrl = utilities._buildHref(hrefParameters, request.data?.query?.context, proxiedValues)
 
-  var headerHost = _.get(request, 'auth.artifacts.host')
+  let headerHost = _.get(request, 'auth.artifacts.host')
   if (_.get(request, 'auth.artifacts.port')) {
     headerHost += ':' + request.auth.artifacts.port
   }
 
-  var headers = {
+  const headers = {
     host: headerHost
   }
   if (request.headers && request.headers.referrer) {
     headers.referrer = request.headers.referrer
   }
 
+  const auth = {
+    ...request.auth
+  }
+  if (!auth.credentials) {
+    auth.credentials = {}
+  }
+  if (!auth.strategy) {
+    auth.strategy = {}
+  }
```

Comment:
I haven't found any documentation for this, but it appears to be necessary now. ???